server/utils.go

```diff
@@ -312,3 +252,21 @@ func checkRelaySignature(bid *spec.VersionedSignedBuilderBid, domain phase0.Doma
 
 	return bls.VerifySignatureBytes(msg[:], sig[:], pubKey[:])
 }
+
+func getPayloadResponseIsEmpty(payload *builderApi.VersionedSubmitBlindedBlockResponse) bool {
+	switch payload.Version {
+	case spec.DataVersionCapella:
+		if payload.Capella == nil || payload.Capella.BlockHash == nilHash {
+			return true
+		}
+	case spec.DataVersionDeneb:
+		if payload.Deneb == nil || payload.Deneb.ExecutionPayload == nil ||
+			payload.Deneb.ExecutionPayload.BlockHash == nilHash ||
+			payload.Deneb.BlobsBundle == nil || payload.Deneb.BlobsBundle.Blobs == nil {
```

Comment:
I don't think it's necessary to check if `Blobs` is `nil` here. It's not a pointer.

<img width="589" alt="image" src="https://github.com/flashbots/mev-boost/assets/95511699/f375c568-8d68-421e-8a02-acbd64a8715e">

IIRC a nil slice will be treated like a zero-length slice.